modules/javafx.graphics/src/main/java/com/sun/javafx/css/BitSet.java

```diff
@@ -220,19 +224,30 @@ public boolean contains(Object o) {
 
         final T t = cast(o);
 
+        if (t == null) {  // if cast failed, it can't be part of this set
```

Comment:
`BitSet` did not conform to the collection contract here.  It can happen in current JavaFX releases by calling a method that returns `Set<PseudoClass>` and asking it `contains("text")` -- as `String` cannot be cast to `PseudoClass`, this will result in a `ClassCastException` while it should just return `false`.